### DISCUSSION TITLE
v3.3.0b8

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#1033] [Announcements]
## Breaking Change
* Turn off by default deprecated send to rooms (you can still enable it until v4) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1032
If you are still using the espresense/rooms/* messages, please move to the espresense/devices/{device}/* messages

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v3.3.0b7...v3.3.0b8

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.3.0b8'>v3.3.0b8</a>.</em>

**Comment by davlaw** [comment]
Some feedback, no reboots so far :)

**Reply by DTTerastar** [reply]
It uses a ton less memory in my testing.
